index.html

```diff
@@ -8018,63 +8018,64 @@ <h3>Graph Containers</h3>
       <pre class="original selected nohighlight" data-transform="updateExample">
       <!--
       {
-        "@context": {
-          "@version": 1.1,
-          "generatedAt": {
-            "@id": "http://www.w3.org/ns/prov#generatedAtTime",
-            "@type": "http://www.w3.org/2001/XMLSchema#date"
+        "@context": "https://w3c.github.io/vc-data-model/vocab/credentials.jsonld",
+        "id": "http://example.edu/credentials/332",
+        "type": "VerifiableCredential",
+        "issuer": "https://example.edu/issuers/4",
+        "issuanceDate": "2017-02-24T00:00:00Z",
+        ****"verifiableCredential"****: {
```

Comment:
```suggestion
        ****"credentialSubject"****: {
```

Reply:
The quads, etc. will also need to change of course.

Reply:
Note that `credentialSubject` is not an `@graph` container. The term `verifiableCredential` is one, however, it is used in a `VerifablePresentation` not a `VerifiableCredential`. That being said, `proof` *is* an `@graph` container ... and since @iherman asked for just one example of a `@graph` container we could leave this as is to address his concern. Otherwise, we should wrap the whole credential in a `VerifiablePresentation` that points to the credential via `verifiableCredential`.

Reply:
Based on @iherman's comments, we probably need to go to an entirely different example, anyway. Or, trim down the example and use an inline context.